tmt/base.py

```diff
@@ -772,12 +774,19 @@ def _fmf_id(self) -> None:
 
     # TODO: cached_property candidates
     @property
-    def fmf_root(self) -> Path:
-        return Path(self.node.root)
+    def fmf_root(self) -> Optional[Path]:
+        # Check if fmf root exists
```

Comment:
Probably "if fmf root is defined", because we don't really check whether it exists, i.e. `Path(self.node.root).exists()`.